.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -1,64 +1,46 @@
-name: 🐛 Bug Report
-description: Report a reproducible bug
-labels: ["bug"]
-projects: ["soujava/3"]
+name: Bug report 🐞
+description: File a bug report
+title: "[Bug] "
+labels: bug
 body:
-  - type: markdown
+  - type: checkboxes
+    id: existing-issue
     attributes:
-      value: |
-        Please provide all the information requested. Issues that do not follow this format are likely to stall.
-        
-        Please test using the latest POM Editor Maven Plugin release to make sure your issue has not already been fixed: https://github.com/soujava/pom-editor-maven-plugin/releases 
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+        - label: I have searched the existing issues
+          required: true
```

Comment:
@Brijeshthummar02 , it's more reasonable to ask the link to the existing issue that we're asking for. If it's provided, so it exists, otherwise, the issuer can just let it blank.